crates/pop-cli/src/commands/up/mod.rs

```diff
@@ -126,6 +134,25 @@ mod tests {
 		cli.verify()
 	}
 
+	#[tokio::test]
+	async fn detects_parachain_correctly() -> anyhow::Result<()> {
+		let temp_dir = tempfile::tempdir()?;
+		let name = "parachain";
+		let project_path = temp_dir.path().join(name);
+		let config = Config {
+			symbol: "DOT".to_string(),
+			decimals: 18,
+			initial_endowment: "1000000".to_string(),
+		};
+		instantiate_template_dir(&Parachain::Standard, &project_path, None, config)?;
```

Comment:
```suggestion
		instantiate_template_dir(&Rollup::Standard, &project_path, None, config)?;
```

Reply:
The struct rename will be addressed once the naming is updated across all commands (e.g., pop new ...). Adding an alias here is possible but unnecessary at this stage.